source/code/plugins/tailfilereader.rb

```diff
@@ -365,7 +395,7 @@ def read_inode
     end
   end.parse!
 
-  a = Tailscript::NewTail.new(ARGV)
+  a = Tailscript::NewTail.new(ARGV[0])
```

Comment:
are we sure all callers to this script pass an array? and not only sudo was passing Array?

Reply:
AFAIK, this plugin itself and its test plugin are the only ones that call this class.
What do you mean by this - "and not only sudo was passing Array?" ?